jib-core/src/main/java/com/google/cloud/tools/jib/registry/credentials/DockerConfig.java

```diff
@@ -106,7 +106,7 @@ DockerCredentialHelper getCredentialHelperFor(String registry) {
   DockerCredentialHelper getCredentialHelperFor(
       DockerCredentialHelperFactory dockerCredentialHelperFactory, String registry) {
     List<Predicate<String>> registryMatchers = getRegistryMatchersFor(registry);
-    Map.Entry<String, AuthTemplate> firstMatchInAuths =
+    Map.Entry<String, ?> firstMatchInAuths =
```

Comment:
Was this intentional?

Reply:
Did not really intend, but I guess we can have it here as a trivial cleanup. Signifies that the value is not being used, and the added benefit of less cramped code.

Reply:
kk sounds good